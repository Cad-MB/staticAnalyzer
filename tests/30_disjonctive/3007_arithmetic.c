{
    int x;
    int y;
    x = 2;
    if (x <= 3) {
      y = x * 2;
    } else {
      y = x + 5;
    }
    x = y - 1;
    print_all;
  }