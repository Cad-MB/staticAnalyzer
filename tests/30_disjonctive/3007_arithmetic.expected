tests/30_disjonctive/3007_arithmetic.c:11.4-14: [ x in {[3;3]}, y in {[4;4]} ]
analysis ended